components/ConfirmDetailsContainer/ConfirmDetailsContainer.jsx

```diff
@@ -0,0 +1,89 @@
+import { Children, useState } from "react";
+import { Text, Box, Flex, Collapse, Spacer, Icon, Button } from "@chakra-ui/react";
+import Pencil from "../../public/images/other/pencil.svg"
+
+export default function ConfirmDetailsContainer({ methodIcon, title, children, describtion }) {
```

Comment:
just a small typo with for one of the arguments

```suggestion
export default function ConfirmDetailsContainer({ methodIcon, title, children, description }) {
```

Reply:
good pick-up, just fixed it in all 3 places 👍